custom_components/zwave_mqtt/light.py

```diff
@@ -0,0 +1,128 @@
+"""Support for Z-Wave lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_TRANSITION,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_TRANSITION,
+    Light,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .entity import ZWaveDeviceEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+# pylint: disable=unnecessary-pass,unused-argument
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
We can remove this, this stuff is all new so there is no need for an "old method".